Types/HttpHeader.php

```diff
@@ -41,9 +41,16 @@ public static function create(array $options, array $tokens) {
         $headers = empty($headers) ? [] : $headers;
         $headers = is_string($headers) ? explode(',', $headers) : $headers;
 
+        // Do not send pagination headers if pagination is sent as query params
```

Comment:
I like to add inline comments where the line isn't immediately readable. Anything with multiple double-negatives usually win a comment from me.